src/core/appconfig.py

```diff
@@ -0,0 +1,24 @@
+
+class AppConfig:
+    __instance = None
+
+    def __init__(self, city, json_file):
+        if AppConfig.__instance is None:
```

Comment:
Valahogy jó lenne az objektum kreálást logika nélkül megoldani.

Reply:
Gondolkodtam ezen egy picit. Ha nekünk ez a config statikus lenne akkor lenne jó a singleton. Viszont mi szeretnénk paraterizált configot kapni tehát itt úgy kellene előáĺlítani a config osztályt ahogyan azt a mostani commitban ia tetted.
Holnap majd személyesen is beszélhetünk erről, de azért is írtam ezt ide mert nen akarom elfelejteni :D